dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/enrollment/AttributeValidator.java

```diff
@@ -103,7 +101,7 @@ public void validate(Reporter reporter, TrackerBundle bundle, Enrollment enrollm
 
         attributeValueMap.put(attribute.getAttribute(), attribute.getValue());
         validateAttributeValue(reporter, enrollment, teAttribute, attribute.getValue());
```

Comment:
Can we get rid of inheritance and move that validation from `validateAttributeValue` into `validateValueType` which actually also validates the value

Reply:
I can remove the inheritance moving `validateAttributeValue` into `ValidationUtils` but I cannot move it into `validateValueType` as it is shared with data values validation as well and `validateAttributeValue` is specific for attributes

Reply:
sounds good 😄